src/test/rustdoc/doc-masked.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(doc_masked)]
+
+#![doc(masked)]
+extern crate std as realstd;
+
+// @has doc_masked/struct.LocalStruct.html
+// @has - '//*[@class="impl"]//code' 'impl LocalTrait for LocalStruct'
+// @!has - '//*[@class="impl"]//code' 'impl Copy for LocalStruct'
```

Comment:
Masking failed. I guess it's because `Copy` and `usize` are from libcore instead of libstd?

```
[00:50:50] ---- [rustdoc] rustdoc/doc-masked.rs stdout ----
[00:50:50] 	
[00:50:50] error: htmldocck failed!
[00:50:50] status: exit code: 1
[00:50:50] command: "/usr/bin/python2.7" "/checkout/src/etc/htmldocck.py" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc/doc-masked.stage2-x86_64-unknown-linux-gnu" "/checkout/src/test/rustdoc/doc-masked.rs"
[00:50:50] stdout:
[00:50:50] ------------------------------------------
[00:50:50] 
[00:50:50] ------------------------------------------
[00:50:50] stderr:
[00:50:50] ------------------------------------------
[00:50:50] 18: @!has check failed
[00:50:50] 	`XPATH PATTERN` did not match
[00:50:50] 	// @!has - '//*[@class="impl"]//code' 'impl Copy for LocalStruct'
[00:50:50] 24: @!has check failed
[00:50:50] 	`XPATH PATTERN` did not match
[00:50:50] 	// @!has - '//*[@id="implementors-list"]//code' 'impl LocalTrait for usize'
[00:50:50] 
[00:50:50] Encountered 2 errors
[00:50:50] 
[00:50:50] ------------------------------------------
[00:50:50] 
[00:50:50] thread '[rustdoc] rustdoc/doc-masked.rs' panicked at 'explicit panic', /checkout/src/tools/compiletest/src/runtest.rs:2511:8
[00:50:50] note: Run with `RUST_BACKTRACE=1` for a backtrace.
[00:50:50] 
[00:50:50] 
[00:50:50] failures:
[00:50:50]     [rustdoc] rustdoc/doc-masked.rs
[00:50:50] 
[00:50:50] test result: FAILED. 162 passed; 1 failed; 1 ignored; 0 measured; 0 filtered out
```